README.md

```diff
@@ -1,38 +1,19 @@
-# STARTER-team01
+# f23-7pm-2-team01
 
 Lab instructions: <https://ucsb-cs156.github.io/f23/lab/team01.html>
 
-* TODO: Update the following table, then delete this line that says "TODO".
+| Name           | GitHub Id        | Service                     | Controller                |
+|----------------|------------------|-----------------------------|---------------------------| 
+| Tiger Yu       | tigeryu8900      | `JokeQueryService`          | `JokeController`          |   
+| Jonathan Cheng | JonathanCheng123 | `LocationQueryService`      | `LocationController`      |   
+| Richard Huang  | huangderful      | `PublicHolidayQueryService` | `PublicHolidayController` |   
+| Hongrui Su     | raysujf          | `TidesQueryService`         | `TidesController`         |   
+| Andy Ouyang    | MuMuKow          | `UniversityQueryService`    | `UniversityController`    |
+| Kyle Wong      | KyleWong288      | `ZipCodeQueryService`       | `ZipCodeController`       |
 
-```
-|   Name    | GitHub Id |  Service                    | Controller                |
-|-----------|-----------|-----------------------------|---------------------------| 
-|           |           | `LocationQueryService`      | `LocationController`      |   
-|           |           | `PublicHolidayQueryService` | `PublicHolidayController` |   
-|           |           | `TidesQueryService`         | `TidesController`         |   
-|           |           | `UniversityQueryService`    | `UniversityController`    |
-|           |           | `ZipCodeQueryService`       | `ZipCodeController`       |
-```
+Repo: https://ucsb-cs156-f23/f23-7pm-2-team01
 
-
-* TODO: Change the title of this README 
-  in the text `# STARTER-team01` above
-  to match the name of your repo, i. e., `f23-5pm-2-team01`, then delete
-  this TODO item.
-
-* TODO: Correct the links to repo below, 
-  then delete this TODO.  Replace it with 
-  a link to your repo, e.g. 
-  https://github.com/ucsb-cs156-f23/f23-5pm-2-team01
-
-Repo: https://ucsb-cs156-f23/STARTER-team01
```

Comment:
Keep the link to the repo, but change it to our repository link

Reply:
The link is actually a line below. I forgot to add the `github.com\` part.